### DISCUSSION TITLE
Welcome to ocsf-schema Discussions!

### DISCUSSION THREAD

**Discussion by rroupski** [ocsf/ocsf-schema#24] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mikeradka** [comment]
Greetings all 👋 , this is Mike Radka from the Splunk team. I am working with Roumen Roupski and Paul Agbabian on our implementation of OCSF over at Splunk, Inc. With this community, I will be contributing to the overall end-to-end schematization for OCSF and its structure, with a deep focus on the following categories:

- System Activity
- Network Activity
- Security Findings
- Configuration/Inventory

Don't be a stranger!

**Comment by josiah-hagen** [comment]
Howdy all (y'all), this is Josiah Hagen from Trend Micro, and I've been asked to check out OCSF progress with an eye on implications for data scientists and semantic integrations.  Perhaps I can even contribute ;P
